apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.py

```
@@ -73,12 +74,20 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
+        pl.get_string_attrib(element, "variables", VARIABLES_DEFAULT)
+    )
+
+    if len(single_variable[0]) != 0 and len(variables) != 0:
+        raise ValueError("variable is deprecated, use only 'variables' property")
+    if len(variables[0]) == 0:
+        variables = single_variable
 
-    if len(variables) > 1:
-        raise ValueError(f"Only one variable is supported for question {name}")
+    if len(variables) > 8:
+        raise ValueError("Too many variables specified.")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)